README.md

```diff
@@ -201,6 +201,10 @@ Once a custom resource is installed, users can create, update and access its obj
   - [BackingStore](doc/backing-store-crd.md) - Storage resources. These storage targets are used to store deduplicated, compressed and encrypted chunks of data.
   - [NamespaceStore](doc/namespace-store-crd.md) - Data resources. These storage targets are used to store and read plain data.
   - [BucketClass](doc/bucket-class-crd.md) - Policies applied to a class of buckets, defines bucket policies relating to data placement.
+  - [Bucket Types](doc/bucket-types.md) - Overview of data and namespace buckets, and supported services
+  - [Bucket Replication](doc/bucket-replication.md) - Overview of bucket replication rules in NooBaa, including log-based optimizations, inner workings, and example rules
+  - [S3 API Compatibility](doc/s3-compatibility.md) - Overview of S3 API compatibility in NooBaa
+  - [AWS API Compatibility](doc/AWS_API_Compatibility.md) - Overview of AWS API calls support in NooBaa
```

Comment:
I think you can move some of them to a higher number in the list as they are more basic:
- [NooBaa]
- [S3 API Compatibility]
- [AWS API Compatibility]
- Customer Resources Definitions ([CRDs]